### DISCUSSION TITLE
Theme update

### DISCUSSION THREAD

**Discussion by marvinmoran** [primefaces/community#1282] [PrimeFaces Templates]
Hello, I have bought several themes for primefaces but it has been a long time since they have updated the themes since I have seen that for primevue, primeng and primereact there are versions of more modern and updated themes, I am willing to pay for them but I would like to have more information if they will be supported

**Comment by melloware** [comment]
I moved this thread to "PrimeFaces Templates" area since its theme related.

**Reply by melloware** [reply]
@marvinmoran when you say "updated" they were recently updated for PF13 if you check the PrimeStore account. Or do you mean no NEW templates have been introduced recently?

**Comment by marvinmoran** [comment]
Yes, I have realized that all templates are already updated to primefaces 13, but my question is more oriented towards improvements in the design of the templates, because I saw that some improved them for primevue, primeng but not for primefaces

**Reply by melloware** [reply]
Can you be specific about what you are looking for?  Specific examples?

**Comment by marvinmoran** [comment]
For example, the verona and apollo template look much better in primeng and in primefaces it does not look the same or not updated
